Resources/Prototypes/Entities/Objects/Specific/Medical/hypospray.yml

```diff
@@ -18,3 +18,47 @@
   - type: ExaminableSolution
     solution: hypospray
   - type: Hypospray
+
+- type: entity
+  name: chemical medipen
+  parent: BaseItem
+  description: A sterile injector for rapid administration of drugs to patients. This one can't be refilled.
+  id: ChemicalMedipen
+  components:
+  - type: Sprite
+    sprite: Objects/Specific/Medical/medipen.rsi
+    layers:
+    - state: medipen
+      map: ["enum.SolutionContainerLayers.Fill"]
```

Comment:
netsync false?

Reply:
is this why the sprite was randomly reverting for no reason sometimes

Reply:
probably